docs/code-snippets/office-snippets.yaml

```
@@ -996,6 +996,15 @@ Office.Context#contentLanguage:member:
     function write(message){
         document.getElementById('message').innerText += message; 
     }
+Office.Context#commerceAllowed:member:
+  - |-
+    // Check if the add-in is running on an iPad.
+    const allowCommerce = Office.context.commerceAllowed;
+    const isTouchEnabled = Office.context.touchEnabled;
+    if (allowCommerce === false && isTouchEnabled === true) {
```

Comparing boolean values like that seems a little redundant.
```suggestion
    if (!allowCommerce && isTouchEnabled) {
```